fiasko_bro/code_validator.py

```diff
@@ -9,6 +10,24 @@ def validate_repo(path_to_repo, path_to_original_repo=None):
     return code_validator.validate(path_to_repo, path_to_original_repo)
 
 
+def py_files_count(path_to_repo):
+    counter = 0
+    for _, _, files in os.walk(path_to_repo):
+        for file in files:
+            if file.endswith('.py'):
+                counter += 1
+    return counter
+
+
+def is_repos_too_large(path_to_repo, path_to_original_repo, max_num):
+    num_of_py_files = py_files_count(path_to_repo)
+    if path_to_original_repo:
+        num_of_py_files_original_repo = py_files_count(path_to_original_repo)
+        return (num_of_py_files_original_repo > max_num or
+                num_of_py_files > max_num)
+    return num_of_py_files > max_num
```

Comment:
This is actually code duplication. To avoid that, first check the main repo (and return if it fails) and then check the origin repo if it's present and return the result.